src/lib/viewports/orthographic-viewport.js

```diff
@@ -13,15 +13,14 @@ export default class OrthographicViewport extends Viewport {
     // projection matrix arguments
     near = 1, // Distance of near clipping plane
     far = 100, // Distance of far clipping plane
-    fovy = 75, // Field of view covered by camera
     left, // Left bound of the frustum
     top, // Top bound of the frustum
     // automatically calculated
     right = null, // Right bound of the frustum
     bottom = null // Bottom bound of the frustum
   }) {
     right = Number.isFinite(right) ? right : left + width;
-    bottom = Number.isFinite(bottom) ? right : top + height;
+    bottom = Number.isFinite(bottom) ? bottom : top + height;
```

Comment:
Ortho viewports were never tested. As I said, the viewport code is still experimental...